docs/cse/onboarding-checklist-cse.md

```diff
@@ -0,0 +1,86 @@
+---
+id: onboarding-checklist-cse
+title: Onboarding checklist for CSE
```

Comment:
Titles get titlecased (H2 and below is sentence case)

```suggestion
title: Onboarding Checklist for CSE
```

Reply:
Fixed by commit "[Updates from review by Kim Pohas](https://github.com/SumoLogic/sumologic-documentation/pull/2009/commits/fd35c19fcc5fb01c9cb0309e3f6d83aeb2dd3d30)".